.openzeppelin/unknown-31337.json

```diff
@@ -0,0 +1,430 @@
+{
```

Comment:
What's the purpose of this file? If it is generated on compilation, it can be put in `.gitignore`

Reply:
it is generated on compilation
I added to .gitignore